repository/b.json

```diff
@@ -32,6 +32,10 @@
 				{
 					"sublime_text": ">=3000",
 					"tags": true
+				},
+				{
+					"sublime_text": ">=4000",
+					"tags": "st4-"
```

Comment:
I haven't seen that convention used before, where both ranges match the ST4 version. Are you sure that Package Control handles it correctly? Typically the ranges would be exclusive (<4000 & >=4000)

Reply:
I actually haven't tested it. It doesn't seem correct to mark the existing versions as incompatible with ST4.

Reply:
But it's not necessarily being defined here which versions (tags) are compatible but rather which versions you want to serve to specific ST versions.

Reply:
I've changed it.